File: Java/Project Euler #1.java
public class ProjectEuler
{
	public static void main(String args[])
	{
		int sum = 0;
		for (int i = 3; i < 1000; i += 3)
			sum += i;
		for (int i = 5; i < 1000; i += 5)
			sum += i;
		for (int i = 15; i < 1000; i += 15)
			sum -= i;
		System.out.println(sum);
	}
}

File: Javascript/Project Euler #1.js
var sum = 0;
for (var i = 3; i < 1000; i += 3)
	sum += i;
for (var j = 5; j < 1000; j += 5)
	sum += j;
for (var b = 15; b < 1000; b += 15)
	sum -= b;
console.log(sum);
